html/changelogs/AutoChangeLog-pr-34338.yml

```diff
@@ -1,4 +1,4 @@
 author: "ShizCalev"
 delete-after: True
 changes: 
-  - bugfix: "Some areas will no longer present themselves at \"ÿUnexplored Area\""
+  - bugfix: "Some areas will no longer present themselves at \"Unexplored Area\""
```

Comment:
at least use the other example, `\improper Unexplored Area` since this changelog will make no sense without that